scripts/run_Kraken2-report.R

```diff
@@ -0,0 +1,75 @@
+# Title  : run Kraken2 report
+# author: mfaxel
```

Comment:
I'm confused about this, because this file was authored by Miriam.  Why is it created as part of this pull request?  (Or is is it added by a commit authored by Miriam?)

Does this pull request depend on another branch?